test/webpack.config.js

```diff
@@ -102,21 +116,34 @@ function getFirstKey(object) {
 
 module.exports = env => {
```

Comment:
I would keep the function how it was originally in this file as `getConfig` make it so that the exported function simply calls and return the object, this way you can log it if you want.